linux_os/guide/services/ssh/package_openssh-clients_installed/rule.yml

```diff
@@ -0,0 +1,26 @@
+documentation_complete: true
+
+prodtype: rhel8
+
+title: 'Install OpenSSH client software'
+
+description: |-
+    {{{ describe_package_install(package="openssh-clients") }}}
+
+rationale: 'The <tt>openssh-clients</tt> package needs to be installed to meet OSPP criteria.'
```

Comment:
Not a great rationale.

Suggestion inspired by package info:
`This package includes the clients necessary to make encrypted connections to SSH servers.`